vademecum_VT.md

```diff
@@ -84,13 +84,13 @@ Con _Notes_ non si avrà più questo problema! Mentre navighi in Firefox basta p
 Finalmente anche in italiano. _Leggi e registra_ frasi o _revisiona_ quelle registrate da altri utenti. Questo progetto punta a realizzare un sistema in grado di comprendere la voce di chiunque e di comprenderne tutte le parole, a prescindere da come esse vangano pronunciate o dall'accento. Parla come lo fai nella vita reale!
 Non ci saranno più problemi di riconoscimento per chi è affetto da _Sigmatismo_ o _Rotacismo_.
 
-#### Servo |QRcode|
-Realizzato col linguaggio di programmazione Rust; è un browser engine ancora agli inizi e che punta a migliorare notevolmente l'utilizzo di memoria e, soprattutto, in termini di sicurezza.
-
 #### MDN |QRcode|
 MDN, _Mozilla Developer Network_, è il servizio dedicato alla programmazione realizzato da Mozilla! Con esso è possibile imparare nuovi linguaggi di programmazione (lato Web), approfondire quelli che già si conoscono, utilizzare script contenuti in esso per vostri progetti o scoprire nuove tecnologie Web, come _HTTP_, _WebExtensions_, e altro ancora.
 Tutto questo semplicemente recandosi sul portale dedicato e a costo zero!
 
+#### Servo |QRcode|
+Servo è un browser engine sperimentale, scritto interamente in *Rust*, che serve per creare e testare i nuovi componenti del motore Gecko di Firefox Quantum, come il parser HTML o Stylo, il parser CSS.L'obiettivo è utilizzare totalmente le potenzialità di Rust migliorando la sicurezza, la modularità e le prestazioni, utilizzando i moderni processori  multicore e la GPU.
```

Comment:
non serve per creare e testare i componenti di firefox quantum.

Reply:
@Mte90 cosa ne pensi della revisione che ho fatto io in merito?

Reply:
Va bene :-)

Reply:
@Mte90 intendo questa:

Servo è un browser engine sperimentale, scritto interamente in Rust.
In futuro, quando sarà stabile e completo, sostituirà l'ormai "anziano" Gecko. In termini di velocità, infatti, risulta essere molto più veloce di Gecko, grazie alle tante nuove tecnologie che si nascondono sotto un unico nome: Servo.
Alcune tecnologie sono implementate anche in Gecko (Stylo ne è un esempio eclatante).

Reply:
Non ho capito. Puoi scrivere proprio il testo in chiaro a scanso di equivoci per favore? xD (sto andando in tilt, scusa)

Reply:
Servo è un browser engine sperimentale, scritto interamente in Rust.
Attualmente vengono utilizzati dei suoi compomenti in Gecko e quando sarà stabile e completo, sostituirà l'ormai "anziano" Gecko. In termini di velocità, infatti, risulta essere molto più lento rispetto a Servo che, dietro questo unico nome, implementa tante nuove tecnologie.

Reply:
Scusa se mi permetto, penso sia meglio così:

Servo è un browser engine sperimentale, scritto interamente in Rust.
Attualmente vengono utilizzati dei suoi compomenti in Gecko e, infatti, quando Servo sarà stabile e completo, sostituirà l'ormai "anziano" Gecko. In termini di velocità, infatti, risulta essere molto più lento rispetto a Servo che, dietro questo unico nome, implementa tante nuove tecnologie.

(era una ripetizione... anche così rimane la ripetizione ma almeno si hanno i soggetti chiari xD)
Che ne dici?

Reply:
ok!